runtime/autoload/provider/script_host.py

```diff
@@ -71,12 +71,14 @@ def teardown(self):
         sys.stderr = self.saved_stderr
 
     @neovim.rpc_export('python_execute', sync=True)
+    @neovim.rpc_export('python3_execute', sync=True)
```

Comment:
> Confirmed the problem.

> @tarruda Can you help the problem?

I'm don't think you can set this decorator multiple times. You can do this though:

```python
@neovim.rpc_export('python3_execute', sync=True)
def python3_execute(self, *args):
    return self.python_execute(*args)

@neovim.rpc_export('python_execute', sync=True)
def python_execute(self, script, range_start, range_stop):
    ...
```